.prettierrc.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  overrides: [
+    {
+      files: '*.{js,ts}',
+      options: {
+        singleQuote: true,
```

Comment:
I can also remove this for now and stick to `"`. Not the default in Ember apps but it would introduce more noise here. I don't really have a preference. As long as it's consistent I'm happy 😄.